pom.xml

```diff
@@ -187,18 +193,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>com.github.ekryd.sortpom</groupId>
-        <artifactId>sortpom-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <goals>
-              <goal>sort</goal>
-            </goals>
-            <phase>verify</phase>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
I understand you removed it because it was duplicated, but we should keep the execution configuration.

Reply:
Added back the plugin